### PR TITLE
Support the prompt option with Meteor.loginWithGoogle

### PR DESCRIPTION
**Pull request by zimme**
This PR allows the use of `prompt` for Google login. It allows for more fine-grained control over which prompt(s) to show.

It keeps backwards compatibility with `forceApprovalPrompt`, but if used together `prompt` takes precedence.
```js
Meteor.loginWithGoogle({prompt: 'select_account'});
```
https://developers.google.com/identity/protocols/OpenIDConnect#prompt

I've split the PR into 3 commits to make it easier to only choose some changes.

CLA should be accepted by both me personally and the company Hansoft

### CODE REVIEW

File: packages/google/google_client.js [hunk 1]
```diff
@@ -44,8 +43,13 @@ Google.requestCredential = function (options, credentialRequestCompleteCallback)
...
         '&redirect_uri=' + OAuth._redirectUri('google', config) +
         '&state=' + OAuth._stateParam(loginStyle, credentialToken) +
-        '&access_type=' + accessType +
-        '&approval_prompt=' + approvalPrompt;
+        '&access_type=' + accessType;
+
+  if (typeof options.prompt === 'string') {
+    loginUrl += '&prompt=' + options.prompt;
+  } else if (options.forceApprovalPrompt) {
+    loginUrl += '&prompt=consent';
```
Comment (stubailo) [inline packages/google/google_client.js]: Doesn't this actually change the behavior when `forceApprovalPrompt` is passed? Before, it used to send `force`, now it sends `consent`?

**Comment by stubailo** [comment]
Comment inline.

File: packages/google/google_client.js (hunk 1 above)
Comment (zimme) [inline packages/google/google_client.js]: `prompt=consent` have the same effect as `approval_prompt=force`

**Comment by zimme** [comment]
Maybe the accounts-ui package can be updated to use prompt too. As I don't use the accounts-ui packages myself I didn't feel I was the right person for the job.

I should be able to look into it if I get an explicit request to do so.

**Comment by stubailo** [comment]
Ah, I guess then it would be added to `Accounts.ui.config`. I don't think it's a top priority, let's wait until someone asks.

**Pull request merged**